### DISCUSSION TITLE
nested dropdown menu

### DISCUSSION THREAD

**Discussion by milahu** [picocss/pico#543] [Ideas]
in [picocss.com/docs/dropdown](https://picocss.com/docs/dropdown)
there is missing an example of a \*nested\* dropdown menu

example of a nested hamburger menu, aka multi-level menu
[codepen.io/dnweij/pen/VwdQKeL](https://codepen.io/dnweij/pen/VwdQKeL) via https://github.com/picocss/pico/issues/62#issuecomment-1322702329

related #116

**Comment by CrlH** [comment]
I accomplished this rather easily with v1 - but haven't found an easy way to do so in v2...

My use case is slightly different, where I have a nested menu in the top `<nav>` element. I did so with minor modifications to the `<li role="list">` syntax that was available for dropdowns in v1, but no longer works in v2.

@lucaslarroche - Any chances you're willing to look into this? Reinstating the `<li>` syntax would be a good start. Happy to share the modifications I made for v1 too. Not a SASS wizard, otherwise I'd start a PR :-)

**Comment by Yohn** [comment]
I’ve added and / or improved upon this feature within my fork of [PicoCSS](https://github.com/Yohn/PicoCSS)

Feel free to try it out and let me know if you run into any issues!
